source/release-notes/backend.txt

```diff
@@ -13,6 +13,39 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240605:
+
+5 June 2024 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Atlas Events
```

Comment:
🤔 what are Atlas Events? Should there be additional information here for context?

Reply:
n/a